code/game/objects/items/storage/secure.dm

```diff
@@ -47,25 +47,24 @@
 				return
 
 			if (isscrewdriver(W))
-				if (do_after(user, 20, TRUE, 5, BUSY_ICON_BUILD))
-					src.open =! src.open
-					user.show_message(text("<span class='notice'> You [] the service panel.</span>", (src.open ? "open" : "close")))
+				if (do_after(user, 20, TRUE, src))
+					open = !open
+					to_chat(user, "<span class='notice'> You [open ? "open" : "close"] the service panel.</span>")
 				return
-			if (ismultitool(W) && (src.open == 1)&& (!src.l_hacking))
-				user.show_message(text("<span class='warning'> Now attempting to reset internal memory, please hold.</span>"), 1)
-				src.l_hacking = 1
-				if (do_after(usr, 100, TRUE, 5, BUSY_ICON_BUILD))
```

Comment:
Is the removal of the icon intended here?

Reply:
sort of sneaky action and no visible message.